### ISSUE TITLE
VAMC Locations: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16964]
## Description
VAMC Location pages need the following updates to components:

Example pages:

- [VAMC Boston Locations](https://www.va.gov/boston-health-care/locations/)
- [VAMC Minneapolis locations](https://www.va.gov/minneapolis-health-care/locations/)

### Elements that are not using components that should be (Backlog)
1. Link
   - Facility names should use the link component
   - ~All "Get Directions" links (Main location, Health clinics, Other locations) need to be updated to use [the v1 component](https://design.va.gov/components/link/)~
2. ~Telephone~
   - ~The main location number and the phone numbers for Health clinics are not using a component and should be updated to use the `aria-describedby` [v1 component](https://design.va.gov/components/telephone#aria-described-by)~
   - ~The numbers under "Other nearby VA locations" are using the proper component but should be updated to use the `aria-describedby` [variant](https://design.va.gov/components/telephone#aria-described-by)~
   - ~Exts in all phone numbers should be parsed correctly within the component~
3. [Back to top component](https://design.va.gov/components/back-to-top)
   - Update 5/3: After determining that this component does not actually have built in intelligence to only show based on page height:
      -  This should still be added, the Locations pages are typically longer so having this on these pages will probably make sense more often than not. 

## Acceptance Criteria
- [ ] Back to top component has been added
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing

**Comment by maxx1128** [comment]
I estimate this ticket at a **3** for moving over a good amount of relatively simple elements over to components.

**Comment by laflannery** [comment]
FYI I removed the Link component from the scope of this ticket because we can't use this in Get Directions links

**Comment by laflannery** [comment]
Telephone update moved to #16148

**Comment by eselkin** [comment]
Probably a 3, but could consider a 5 given that it has two PRs and updates a React component
Updating `src/site/includes/facilityListing.drupal.liquid` 
Also updating widget "other-facility-locations-list"

**Comment by laflannery** [comment]
@eselkin If this is 2 PRs and it would make more sense to break up this ticket into the individual pieces for each of those PRs we can do that as well, if it makes more sense

**Comment by eselkin** [comment]
It's one display so keep as one ticket @laflannery

**Comment by randimays** [comment]
A design tweak was merged in for this ticket this morning. We missed the deployment window, though, so this'll be in production tomorrow around lunchtime.

**Comment by randimays** [comment]
I thought we missed the deployment window but maybe the deployment lagged today? Our design tweak is in production and has been validated successfully. Closing!
